Server/src/RedisManager.h

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2024 Microsoft, Inc. and others.  All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ */
+
+#ifndef REDISMANAGER_H_
+#define REDISMANAGER_H_
+
+#include <sonic-swss-common/common/dbconnector.h>
```

Comment:
>sonic-swss-common/ [](http://example.com/codeflow?start=10&length=18)

You can simplify these paths. You may refer sonic-swss repo

Reply:
Prefer `#include "common/dbconnector.h"`

Reply:
I see you changed them to commented code. So suggest just remove dead code, not to leave commented code.

Reply:
updated